### PR TITLE
xGov-180: Thurstober Digital Studios - Open Source, No Code Tools

### PR DESCRIPTION
**Pull request by LoafPickleWW**
An overview of the xGov Proposal for Thurstober Digital Studios.

### CODE REVIEW

**Comment by Vidhyanandcs** [comment]
Seems like the proposal number should be 180

**Comment by nonfungibleab** [comment]
Hey @LoafPickleWW your proposal number should match the PR number. Would you please rename it (and on the forum as well if you've created your thread)? Thanks.

**Comment by LoafPickleWW** [comment]
> Hey @LoafPickleWW your proposal number should match the PR number. Would you please rename it (and on the forum as well if you've created your thread)? Thanks.

Done

File: Proposals/xGov-179.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+This new tool will allow for users to select the ARC3/19/69, enter the IPFS CID for the collection, Name with option for starting #, Unit Name with option for starting #, have toggles for freeze and CB, Description, External URL, Creator, Token, and Royalty. They can then upload a sheet with just the traits to mint fast and easy. 
+This Tool Charges 0.1A to Mint. 
+
+Item 7) Tools Homepage and The Laboratory Redesign (New Site/Page Upgrade)
+We want to help consolodate all the information from the No-Code tools we have built and have everything in one place for anyone new to the ecosystem to discover.
+This would include a new tools.thurstober.com website that links to the labs (tools), we (wallet enhancer), and claims subdomains. 
+In addition, this will upgrade the theming of the Labs webpage to match Wallet Enhancer to create a new uniform, upgraded look across all our platforms. 
+
+### Deliverable
+Proposals must include clear, non-technical descriptions of deliverables. We encourage the use of multimedia (blog/video) to help explain your proposal's benefits to the community.
```
[the remark this answers is not in the document]
Comment (LoafPickleWW) [reply Proposals/xGov-179.md]: Done. We link all the tools to https://thurstober.com/

**Comment by LoafPickleWW** [comment]
Deliverables: 

The Deliverables for this proposal are now complete, you can see them here:
Item 1: [The Laboratory 1](https://labs.thurstober.com/vault-send)
Item 2: [https://swapshop.thurstober.com/ 1](https://swapshop.thurstober.com/)
Item 3: [The Laboratory 1](https://labs.thurstober.com/simple-mint) and [The Laboratory 1](https://labs.thurstober.com/simple-update)
Item 4: [The Laboratory 1](https://labs.thurstober.com/multimint-asset-holders)
Item 5: [Wallet Enhancer 1](https://we.thurstober.com/asset/2168901496)
Item 6: [The Laboratory 1](https://labs.thurstober.com/simple-batch-mint)
Item 7: [https://labs.thurstober.com/ 1](https://labs.thurstober.com/)

Repos:
Labs: [GitHub - thurstober-digital/evil-tools](https://github.com/thurstober-digital/evil-tools)
Wallet Enhancer: [GitHub - algovado/wallet-enhancer: Make it easier to interact with the assets (or NFTs) of Algorand.](https://github.com/algovado/wallet-enhancer)
Swap Shop: [GitHub - algovado/swap-shop 1](https://github.com/algovado/swap-shop)

**Comment by LoafPickleWW** [comment]
Please note that all of these sites have now moved to wen.tools. Redirects should work, but all the repos can be found here: https://github.com/LoafPickleWW

**Comment by LoafPickleWW** [comment]
Fixed links:

1) https://www.wen.tools/vault-send
2) https://swap.wen.tools/
3) https://www.wen.tools/simple-mint and https://www.wen.tools/simple-update
4) https://www.wen.tools/multimint-asset-holders
5) https://wallet.wen.tools/asset/392891258
6) https://www.wen.tools/simple-batch-mint
7) https://www.wen.tools/ and https://wallet.wen.tools/ and https://swap.wen.tools/

**Pull request merged**